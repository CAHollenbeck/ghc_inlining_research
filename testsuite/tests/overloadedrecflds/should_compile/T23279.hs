{-# LANGUAGE DuplicateRecordFields #-}

module T23279 where

import T23279_aux

bar = Bar { x = 3, y = 'x', z = False, w = 17.28 }
baz = Baz { z = 1.1 }

v = w
