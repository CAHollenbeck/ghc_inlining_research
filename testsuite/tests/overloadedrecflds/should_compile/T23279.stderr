
T23279.hs:7:13: warning: [GHC-68441] [-Wdeprecations (in -Wextended-warnings)]
    In the use of record field of Bar ‘x’ (imported from T23279_aux):
    Deprecated: "Don't use x"

T23279.hs:7:29: warning: [GHC-68441] [-Wdeprecations (in -Wextended-warnings)]
    In the use of record field of Bar ‘z’ (imported from T23279_aux):
    Deprecated: "Don't use z"

T23279.hs:7:40: warning: [GHC-68441] [-Wdeprecations (in -Wextended-warnings)]
    In the use of record field of Bar ‘w’ (imported from T23279_aux):
    Deprecated: "Don't use w"

T23279.hs:8:13: warning: [GHC-68441] [-Wdeprecations (in -Wextended-warnings)]
    In the use of record field of Baz ‘z’ (imported from T23279_aux):
    Deprecated: "Don't use z"

T23279.hs:10:5: warning: [GHC-68441] [-Wdeprecations (in -Wextended-warnings)]
    In the use of ‘w’ (imported from T23279_aux):
    Deprecated: "Don't use w"
